keyboards/protokeeb/rev1/mcuconf.h

```diff
@@ -0,0 +1,26 @@
+/* Protokeeb Copyright 2024 A-Tech Officials (@atechofficials)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. 
+ * If not, see <https://www.gnu.org/licenses/gpl-3.0.html>.
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+/* RP2040 I2C Driver Selection */
+#undef RP_I2C_USE_I2C0
+#define RP_I2C_USE_I2C0 TRUE
```

Comment:
What is this needed for? This keyboard does not have an I2C peripherals.

Reply:
I previously left these lines of code while testing an I2C OLED display module with the keyboard. I will remove them now that the I2C interface is no longer necessary.